src/plugin/fieldViews/AttributeFieldView.ts

```diff
@@ -6,8 +6,8 @@ import { FieldType } from "./FieldView";
 /**
  * A FieldView representing a node that contains fields that are updated atomically.
  */
-export abstract class AttributeFieldView<Fields extends unknown>
-  implements FieldView<Fields> {
+export abstract class AttributeFieldView<Value extends unknown>
+  implements FieldView<Value> {
```

Comment:
I think this makes it much clearer to devs working on this. The use of field is used widely and specifying its `value` makes a lot of sense.